index.html

```diff
@@ -43,6 +43,139 @@ <h3 class="title dtitle">I'm a software developer.</h3>
                 </div> 
             </section>
         </div>
+
+        <section class="recent-works">
+            <div class="recent-works-header">
+                <h2 class="title">My Recent works</h2>
+                <hr class="recent-works-line">
+            </div>
+
+            <div class="cards">
+                <div class="card">
+                    <div class="card-image"></div>
+                    <div class="card-text">
+                        <p class="card-title">Multi-Post Stories Gain + Glory</p>
```

Comment:
- [ ] please make use of one of the various headings asides the `<h1>` (which is to be used once in the web page). To display the title of the projects you can select from `<h2>, <h3>` etc :+1: